### PR TITLE
Adding meeting notes for Editors meeting 42

### PR DESCRIPTION
**Pull request by mangelsjover**

### CODE REVIEW

**Review by rphair — APPROVED** [review]
I read through these in the other PR, so if they're still the same then they're still fine with me 😎 ... thanks @mangelsjover for cleaning up the commits from previous work 🤩

**Comment by mangelsjover** [comment]
@rphair all the merits are for @KtorZ, he did an amazing job walking me through all the steps!

File: BiweeklyMeetings/2022-04-05.md [hunk 1]
```diff
@@ -0,0 +1,1538 @@
...
+
+
+
+
+**Sebastian Guillemot** - Yeah.
+
+
+
+
+**Matthias Benkort** - The point of having a CIP would be to, well, have cost compatibility or anti rob between wallets. So if it is something Daedalus-specific, then [crosstalk 00:01:56].
```
Comment (KtorZ) [inline BiweeklyMeetings/2022-04-05.md:146]:
```suggestion
**Matthias Benkort** - The point of having a CIP would be to, well, have cost compatibility or interop between wallets. So if it is something Daedalus-specific, then [crosstalk 00:01:56].
```

File: BiweeklyMeetings/2022-04-05.md [hunk 2]
```diff
@@ -0,0 +1,1538 @@
...
+
+
+
+
+**Sebastian Guillemot** - What they can't standardize is this extension to the web Cardano stuff.
```
Comment (KtorZ) [inline BiweeklyMeetings/2022-04-05.md:151]:
```suggestion
**Sebastian Guillemot** - What they can standardize is this extension to the web Cardano stuff.
```

File: BiweeklyMeetings/2022-04-05.md [hunk 3]
```diff
@@ -0,0 +1,1538 @@
...
+
+
+
+
+**Matthias Benkort** - You don't care much what's the pre images about so long as it's a valid credential. Maybe a Plutus script, native script or a vkey.
+
+
+
+
+**Sebastian Guillemot** - Yeah. So I think from the actual CDL specification from the CIP perspective, this is not a hard change. Someone just needs to make it like some sort of or with a tag. The question is just whether or not they want to make that change now or later. I suggest we at least make it now so we have backwards compatibility, and then I'll see this is something for wallets to add support and something for the Catalyst team as well to add support when they scan the chain.
```
Comment (KtorZ) [inline BiweeklyMeetings/2022-04-05.md:474]:
```suggestion
**Sebastian Guillemot** - Yeah. So I think from the actual CDDL specification from the CIP perspective, this is not a hard change. Someone just needs to make it like some sort of or with a tag. The question is just whether or not they want to make that change now or later. I suggest we at least make it now so we have backwards compatibility, and then I'll see this is something for wallets to add support and something for the Catalyst team as well to add support when they scan the chain.
```

**Review by KtorZ — APPROVED** [review]
Approving, modulo small updates :+1:

**Pull request merged**